sdk/storage/Azure.Storage.Common/src/Shared/Constants.cs

```diff
@@ -391,6 +391,8 @@ internal static class DataLake
             public const string GroupHeaderName = "x-ms-group";
 
             public const string PermissionsHeaderName = "x-ms-permissions";
+
+            public const string ACLHeaderName = "x-ms-acl";
```

Comment:
NIT - even for acronyms it would still be camel case, so "AclHeaderName" is better